pkg/daemon/client.go

```diff
@@ -262,9 +262,17 @@ func (c *nydusdClient) BindBlob(daemonConfig string) error {
 	return c.request(http.MethodPut, url, bytes.NewBuffer([]byte(daemonConfig)), nil)
 }
 
-func (c *nydusdClient) UnbindBlob(domainID string) error {
+func (c *nydusdClient) UnbindBlob(domainID, blobID string) error {
 	query := query{}
-	query.Add("domain_id", domainID)
+	if domainID != "" {
+		query.Add("domain_id", domainID)
+		if domainID != blobID {
+			query.Add("blob_id", blobID)
+		}
+	} else {
+		query.Add("blob_id", blobID)
```

Comment:
Can we have a comment describing the blob id and domain id combination here or a UT test? Otherwise, it is hard to maintain

Reply:
Yeah , will add comment after the API fixed

Reply:
As the nydusd API was fixed and merged, shall we have a description comments now?

Reply:
We have to consider the API compatibility here. Old nydusd won't extract "blob_id' from the request, right? So old nydusd still can work with this nydus-snapshotter?

Reply:
Already add some comment for the API , and this new semantic is just used to delete fscache cache files, if the old nydusd did not support this then snapshotter will just ignore the error and print error info, which is as expectation. so I think we do not need compatible with an old nydusd.